src/frontend/tests/features/utils/helpers.ts

```diff
@@ -52,9 +52,11 @@ export async function tryUntil(
 
       break;
     } catch (error) {
-      retry += 1;
-      savedError = error;
-      await page.waitForTimeout(WAIT_FOR_MS);
+      if (error instanceof Error) {
```

Comment:
for my own education, what prompted the need to add this if condition? Thank you!

Reply:
So, with my new node package install, I wound up pulling in a more recent version of TS. They changed the implied Type of errors to `unknown`: https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables

Apparently -- I didn't know this before -- you can throw anything as an "error" in JavaScript. They used to catch them as `any`, but with the new version they changed it to `unknown`. Because `savedError` is explicitly an Error where it first gets created (above the try/catch), the `savedError = error` was failing because it was assigning an `unknown` now with the new TS version. So I put in a bit of type narrowing to guarantee to TS that it was in fact an Error.

More generally, this made TS happy here, but I think it's probably not The One True Pattern. I'm still thinking about it, but I feel like the way to go generally is
```
try {
  ...
} catch(err) {
  if (err instanceof Error) { // Or whatever kind of precise thing you should be catching
    ...
  } else { // Wait, it wasn't an Error we just caught? What the heck is it?
    throw err; // If it's not the Error we expected, we should probably blow up now.
  }
}
```

Reply:
Ahh thank you for the information! That makes sense 🎉